src/analytics/Trace.tsx

```diff
@@ -44,7 +44,8 @@ export const Trace = memo(
 
     useEffect(() => {
       if (shouldLogImpression) {
-        sendAnalyticsEvent(name ?? EventName.PAGE_VIEWED, { ...combinedProps, ...properties })
+        const origin = window.location.origin
```

Comment:
should it be on all events or just page viewed

Reply:
just page viewed. other events can refer back to this one.